Update condition.markdown (#40213)
home-assistant/home-assistant.github.io: source/_docs/automation/condition.markdown

description: "Automations can test conditions when invoked."
---

> Conditions are an optional part of an automation rule. They can be used to prevent the automation's actions from being run. After a {% term trigger %} occurred, all conditions will be checked. The automation will be executed if all conditions return true, otherwise, if any of them does not return true, the automation not start.

Reviewer: ```suggestion
Conditions are an optional part of an automation rule. They can be used to prevent the automation's actions from being run. After a {% term trigger %} occurred, all conditions will be checked. The automation will be executed if all conditions return `true`. If any of the conditions returns `false`, the automation won't start.
```
